Makefile

```diff
@@ -421,6 +421,13 @@ clients/cmd/docker-driver/docker-driver:
 	CGO_ENABLED=0 go build $(GO_FLAGS) -o $@ ./$(@D)
 
 docker-driver-push: docker-driver
+ifndef DOCKER_PASSWORD
```

Comment:
This requires the docker username and password to be defined as environment variables? These credentials are coming from the vault right?
https://github.com/grafana/loki/blob/main/.drone/drone.jsonnet#L36-L37